docs/bin/generate_man.py

```diff
@@ -116,7 +116,7 @@ def opts_docs(cli_class_name, cli_module_name):
     # instantiate each cli and ask its options
     cli_klass = getattr(__import__("ansible.cli.%s" % cli_module_name,
                                    fromlist=[cli_class_name]), cli_class_name)
-    cli = cli_klass([])
+    cli = cli_klass(['opts_docs'])
```

Comment:
@abadger I'm not sure if this will effect the docs build.

Reply:
@samdoran It probably won't but you could do this:
``` python
cli = cli_klass = ([cli_name])
```

which is probably "more right" ;-)

Reply:
This line is likely not needed with your code and appropriately setting cli as well: https://github.com/ansible/ansible/pull/55728/files/d6b433e03bb2648077cd83ae9352011c735afbc3..4fb108769a560f8f0ea9b6c33854884c2bb12120#diff-8c656f386b78148720508ca0dab12b82R127